### PR TITLE
Feature branch of Mint Signup Page

### PR DESCRIPTION
**Pull request by defoebrand**
In this feature branch we created the entirety of our Inuit Mint Signup page.

It involved a header, a form, and a footer.

The form should have input tags matching their purposes and connect to their respective labels.

This HTML has been validated.

### CODE REVIEW

**Comment by voscarmv** [comment]
# Pull Request Under Review by Oscar Mier (TSE) 👨‍💻 🐦

File: index.html [hunk 1]
```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta name="author" content="Brandon and Tiago">
+  <link rel="icon" type="image/png" href="images/intuit-icon.png">
+  <link rel="apple-touch-icon" type="image/png" href="images/intuit-icon.png"/><!--Apple Compatibility1-->
+  <title>Mint Sign Up Clone</title>
+  <!--     <link href="styles/reset.css" rel="stylesheet" type="text/css">   use to erase preset styles as debug tool -->
```
Comment (voscarmv) [inline index.html]: Please remove all comments from your `index.html` file. This is required by Microverse to approve the project, and it makes your code look clean! ✨

File: styles/style.css [hunk 2]
```diff
@@ -0,0 +1,205 @@
...
+  font-family: arial, sans-serif;
+  color: #4c4c4c;
+}
+
+a {
+  text-decoration: none;
+  color: #0077c5;
+}
+
+/* HEADER SECTION */
```
Comment (voscarmv) [inline styles/style.css]: Please remove all comments from your CSS code. This is required by Microverse to approve the project, and it makes your code look clean! ✨

**Review by voscarmv — CHANGES_REQUESTED** [review]
# Impressive work so far team 🧐 💻 

## However, there are still some changes required... 📝 

- Substitute all `id` styling attributes for `class` on your `html` tags.
- Use the semantic `<main>` tag to make your code more easily readable to `web crawlers` 🕷️ 🕸️ 

### Don't forget to check my comments on your code for further required changes 👀 

### And that's it! 😃 👍 Apply the changes and re-submit for review!

## Questions? Comments? Contact me on [twitter](www.twitter.com/voscarmv) or [LinkedIn](www.linkedin.com/in/oscar-mier) 🧑‍💻 🗨️ 🧑‍💻 

### Happy coding! 💻 ⌨️ 🧠

**Comment by defoebrand** [comment]
all instances of ids have been replaced with classes and the form element has been wrapped in a main element.  all comments have been removed

**Review by peterrobert — CHANGES_REQUESTED** [review]
### CHANGES REQUIRED :recycle: 
Very impressive work so far everything in your project is perfect. The website looks exactly like the clone :+1: :+1: the commit messages are perfect :+1: the repo looks professional :+1:  However there is one thing that has prevented me from approving this project. Your HTML has failed to pass validation. Kindly fix the following errors :arrow_down: 
validator-html

And also the CSS file :arrow_down: 
validator-css

Kindly fix those errors and resubmit your project

**HAPPY CODING ** :computer:

**Comment by tiagoes** [comment]
We correct the bugs presented in Html validator and in the Css lints

**Comment by defoebrand** [comment]
Updated CSS to reflect needed changes

**Comment by andresporras3423** [comment]
### **Pull Request Under Review By Oscar Russi (TSE)**

**Review by andresporras3423 — APPROVED** [review]
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:
**Hello @defoebrand and @tiagoes** 
Amazing job :clap: :clap: you have passed all the requirements for this project
Now you are ready to continue with your next challenge :muscle: 
Don't forget to merge with the master branch

Keep going like this,

Oscar Andrés Russi Porras

**Pull request merged**